numpy/polynomial/tests/test_classes.py

```diff
@@ -19,41 +20,48 @@
 
 classes = (
     Polynomial, Legendre, Chebyshev, Laguerre,
-    Hermite, HermiteE)
-
-
-def test_class_methods():
-    for Poly1 in classes:
-        for Poly2 in classes:
-            yield check_conversion, Poly1, Poly2
-            yield check_cast, Poly1, Poly2
-    for Poly in classes:
-        yield check_call, Poly
-        yield check_identity, Poly
-        yield check_basis, Poly
-        yield check_fromroots, Poly
-        yield check_fit, Poly
-        yield check_equal, Poly
-        yield check_not_equal, Poly
-        yield check_add, Poly
-        yield check_sub, Poly
-        yield check_mul, Poly
-        yield check_floordiv, Poly
-        yield check_truediv, Poly
-        yield check_mod, Poly
-        yield check_divmod, Poly
-        yield check_pow, Poly
-        yield check_integ, Poly
-        yield check_deriv, Poly
-        yield check_roots, Poly
-        yield check_linspace, Poly
-        yield check_mapparms, Poly
-        yield check_degree, Poly
-        yield check_copy, Poly
-        yield check_cutdeg, Poly
-        yield check_truncate, Poly
-        yield check_trim, Poly
-        yield check_ufunc_override, Poly
+    Hermite, HermiteE
+    )
+classids = (
+    'Polynomial', 'Legendre', 'Chebyshev', 'Laguerre',
+    'Hermite', 'HermiteE'
+    )
+
+
+
+
+#def test_class_methods():
```

Comment:
To be done? Shouldn't have commented out code left in files...

Reply:
Looks like this is done, and just needs removing.